csi.proto

```diff
@@ -812,6 +834,12 @@ message ListVolumesResponse {
     // published_node_ids MAY include nodes not published to or
     // reported by the SP. The CO MUST be resilient to that.
     repeated string published_node_ids = 1;
+
+    // volume_health shows error conditions reported by the SP.
+    // This field MUST be specified if the
+    // LIST_VOLUMES_VOLUME_HEALTH controller capability is supported.
+    // This field is OPTIONAL.
+    repeated VolumeHealth volume_health = 2 [(alpha_field) = true];
```

Comment:
I am not clear why this field is `repeated`.  Is it a list of heath statuses?  If so, how should the CO interpret multiple statuses for the same volume?  Can we clarify the semantics, or make it not `repeated`?

Reply:
This was made "repeated" before we introduced "Condition" in case there are different errors happening at the same time.  I think we don't need it any more.

@msau42 are you okay if we get rid of "repeated" here?

Reply:
Removed "repeated".

Reply:
What happens if a volume has multiple errors with different conditions?

Reply:
Condition should be a summary state.  If there are multiple errors, driver needs to summarize them and report just one Condition.

I wonder if we should have an array to store multiple errors.

Reply:
That makes sense to me.  A single volume health with a repeated set of `error_code` and `message` fields would allow the SP to report multiple errors without requiring the CO to reason about multiple `condition`s.

Reply:
Ok.  Changed "error_code" to "repeated string".

Reply:
I am wondering if perhaps it would be better to put "error_code" and "message" into a separate message type, and make that thing repeated?  Otherwise, I think we would need to add more clarification to the spec to describe how the CO should correlate error_codes and messages.  For example, if the SP returns a list of 4 error codes, but only 2 messages, how does the CO know which code goes with which message?

Reply:
Yeah, it's better to have a separate message type for that.